.coder_resources/_unused/localisation/not_used_strings.csv

```diff
@@ -813,4 +813,49 @@ blood_rituals;Hold Blood Rituals;;Blutrituale abhalten;;;;;;;;;;;x
 blood_rituals_desc;Hakkar needs more blood! Kill the prisoners! Blood for the blood god!;;Hakkar braucht mehr Blut! T�tet die Gefangenen! Blut f�r den Blutgott!;;;;;;;;;;;x
 hakkar_blessing_1;Hakkar's blessing I;B�n�diction d'Hakkar I;Hakkars Segen I;;;;;;;;;;;x
 hakkar_blessing_2;Hakkar's blessing II;B�n�diction d'Hakkar II;Hakkars Segen II;;;;;;;;;;;x
-hakkar_blessing_3;Hakkar's blessing III;B�n�diction d'Hakkar III;Hakkars Segen III;;;;;;;;;;;x
\ No newline at end of file
+hakkar_blessing_3;Hakkar's blessing III;B�n�diction d'Hakkar III;Hakkars Segen III;;;;;;;;;;;x
+noblegarden;Hold the Noblegarden Festival;Organise la f�te du Jardin des nobles;Haltet das Fest von Noblegarden ab;;;;;;;;;;;x
+lunarfestival;Hold the Lunar Festival;Organise la f�te Lunaire;Haltet das Mondfest ab;;;;;;;;;;;x
+loveisintheair;Hold the Love is in the Air Festival;Organise le festival de l'Amour dans l'air;Haltet das Liebe-in-der-Luft-Fest ab;;;;;;;;;;;x
+childrensweek;Hold the Children's Week Festival;Organise la semaine des enfants;Haltet das Fest der Kinderwoche ab;;;;;;;;;;;x
+midsummerfirefestival;Hold the Midsummer Fire Festival;Organise la f�te du Feu du solstice d'�t�;Haltet das Mittsommer-Feuer-Fest ab;;;;;;;;;;;x
+harvestfestival;Hold the Harvest Festival;Organise la f�te des Moissons;Haltet das Erntedankfest ab;;;;;;;;;;;x
+brewfest;Hold Brewfest;Organise la f�te des Brasseurs;Haltet das Braufest ab;;;;;;;;;;;x
+hallowsend;Hold the Hallows End Festival;Organise la Sanssaint;Haltet das Heiligenfest ab;;;;;;;;;;;x
+pilgrimsbounty;Hold the Pilgrim's Bounty Festival;Organise la festival des Bienfaits du p�lerin;Haltet das Pilgerspendenfest ab;;;;;;;;;;;x
+feastofwinterveil;Hold the Festival of Winter Veil;Organise le festival du Voile d'hiver;Haltet das Winterhauchfest ab;;;;;;;;;;;x
+noblegarden_DESC;The great festival of Noblegarden has long been celebrated by humans, and more recently by the other races of the Alliance. During this joyous event, it is customary for the nobles and lords from each race to hide coins, candy, and the occasional treasures within special eggs painted to look like wildflowers. These eggs are then scattered around major cities for the citizenry to find.;;;;;;;;;;;;;x
+lunarfestival_DESC;Every year, the druids of Moonglade hold a celebration of their city's great triumph over an ancient evil. During the Lunar Festival, citizens of Azeroth have the opportunity to honour their elders by celebrating their sage wisdom, sharing in magnificent feasts, and of course by shooting fireworks!;;;;;;;;;;;;;x
```

Comment:
I believe the first by would still affect the `shooting fireworks` making the latter by unnecessary.
```suggestion
lunarfestival_DESC;Every year, the druids of Moonglade hold a celebration of their city's great triumph over an ancient evil. During the Lunar Festival, citizens of Azeroth have the opportunity to honour their elders by celebrating their sage wisdom, sharing in magnificent feasts, and of course, shooting fireworks!;;;;;;;;;;;;;x
```

Reply:
unnecessary, see the name of the file? these lines were moved to not used

Reply:
Ahh missed that. Sorry.